### DISCUSSION TITLE
[Bug] About the major bug of pool protocol

### DISCUSSION THREAD

**Discussion by NTOHPN** [Chia-Network/chia-blockchain#16470] [Support]
### What happened?

For example,
One day in the future, the chia chain will be fully utilized, and the frequent trading GAS will become very high.
1.75 this part of the assumption has been halved 10 times, and now the reward is 0.175.
This part also assumes that it has been halved 10 times and is now 0.025.
The average fee for farmers with the block is assumed to be 5, which is given to the miners along with 0.025, and 0.175 is allocated to the pool.
At this time, the focus of mining profit of the chia chain is Gas.
However, under the current mine pool agreement, the mine pool cannot allocate Gas, so miners with small calculations have to withdraw from mining activities, because small miners cannot get Gas or share Gas equally through the mine pool for heating.
This is a hard wound in the design of the mine pool protocol.
I hope that the authorities will pay attention to and update the mining pool agreement, which many friends raised when they inspected this chain.

This question has been responded to @ Bernard many times by the Chinese community, but there is no response. It is suggested that Chia consider whether to continue to use @ Bernard? This manager only hangs out with @ Hema to sell NFT, and there is no feedback on the technical problems in the Chinese area, so he promotes it privately for @ Hema!

### Version

all

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by shaneo257** [comment]
hi, most pools operate where the farmer reward (1/8th of the block reward) plus any fees will go to the farmer that farmed that block, then the remaining pool reward (7/8th's of the block reward) goes to the pool
currently because the transaction volume is low, allot of transactions can be completed with no fee, but this is expected to change in the future when the transaction volume increases
my recommendation if if the pool that you are farming in is not giving you farmer rewards plus any fees, would be to change to a different pool.

**Reply by NTOHPN** [reply]
你大概没有看懂我描述的是什么
You probably don't understand what I'm describing.

**Comment by NTOHPN** [comment]
官方之所以移动我这个帖子，官方也没有看懂我描述的是什么！
The official reason why I moved this post, the official did not understand what I described!

**Comment by NTOHPN** [comment]
讲的是池协议存在不可持续的bug
It's about pooling protocols with unsustainable bug.

**Comment by BrandtH22** [comment]
Hey @Ntohpn can you better describe the bug itself?
It sounds like this is more of an undesired design rather than a bug - the fees are designed to go to the farmer and not the pool.

Would you like to change this into a feature request rather than a bug report?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
